.eleventy.js

```diff
@@ -197,6 +197,25 @@ module.exports = function (eleventyConfig) {
 
   // Browsersync Overrides
   eleventyConfig.setBrowserSyncConfig({
+    middleware: function (req, res, next) {
+      const url = new URL(req.originalUrl, `http://${req.headers.host}/)`);
```

Comment:
Please add a comment as to what this all does (just an overview)

Reply:
Ok. Done